src/Writer/PlainFileWriter.php

```diff
@@ -37,7 +37,10 @@ public function finish(): void
     {
         \assert($this->file !== null);
 
-        fclose($this->file);
+        $x = $this->file;
```

Comment:
Why do we need to do it?

Reply:
The fix is better to be similar to what's in `TempFileGZIPWriter`.

Reply:
Yes, sorry, I forgot about this - now done.